<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Excel文件读取并转换Json格式</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/new/prefix1/res/里定图标.png</normaloff>:/new/prefix1/res/里定图标.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelExcelPath">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtnOpenExcelfile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>打开Excel文件</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>391</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowserWenben">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>70</y>
      <width>391</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowserWenben2Json">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>320</y>
      <width>391</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowserExcel2Json">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>320</y>
      <width>391</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>391</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Excel文件数据显示区</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>50</y>
      <width>391</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>文本文件数据显示区</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>391</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Excel文件转Json数据显示区</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>300</y>
      <width>391</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>文本文件转Json数据显示区</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtnOpenWenbenfile">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>0</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>打开文本文件</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelWenbenPath">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
